packages/@okta/vuepress-site/docs/guides/sign-in-to-spa-embedded-widget/main/vue/basic-sign-in.md

```diff
@@ -0,0 +1,196 @@
+Use the [Vue Router](https://router.vuejs.org/) and the [Okta Vue SDK](https://github.com/okta/okta-vue) libraries to simplify your component and route definitions.  Some routes require authentication in order to render and other don't. The following are some basic routes you need to configure for your app:
+
+* A [default page](#default-page-route) to handle basic control of the app.
+* A [login route](#login-route) to show the Sign-In Widget.
+* A [callback route](#callback-route) to parse tokens after a redirect from Okta.
+* A [protected route](#protected-route) for authenticated users to access protected content.
+
+#### Default page route
+
+To create the default `/index` page, update the `src/App.vue` file to provide links to relevant locations in your app. You need to provide a `Login` link to render the Sign-In Widget, a `Logout` link to sign-out of your authenticated session, and links to authenticated pages by using the `authState` property (see [`authStateManager` in Auth JS SDK](https://github.com/okta/okta-auth-js#authstatemanager)).
+
+You can use the following condition to hide or show specific elements:
+
+`v-if="authState && !authState.isAuthenticated"`
+
+See the following `src/App.vue` file example:
+
+```html
+<template>
+  <div id="nav">
+    <router-link to="/">Home</router-link> |
+    <router-link to="/about">About</router-link> |
+    <router-link to="/login" v-if="authState && !authState.isAuthenticated">Login</router-link>
+    <router-link to="/profile" v-if="authState && authState.isAuthenticated">Protected Profile<router-link>
+    <button v-if="authState && authState.isAuthenticated" v-on:click="logout()">Logout</button>
+  </div>
+  <router-view/>
+</template>
+
+<script>
+export default {
+  name: 'app',
+  methods: {
+    async logout () {
+      const publicPath = this.$route.href.replace(new RegExp(this.$route.fullPath + '$'), '');
+      await this.$auth.signOut({ postLogoutRedirectUri: `${window.location.origin}${publicPath}` })
```

Comment:
nit: `postLogoutRedirectUri ` won't be needed for this guide, it's added to handle heroku deployment path.

you can just use
```
async logout () {
  await this.$auth.signOut()
}
```